chart/elastalert/README.md

```diff
@@ -70,8 +70,9 @@ The command removes all the Kubernetes components associated with the chart and
 | `elasticsearch.certsVolumeMounts`            | mount certs volumes, required to mount ssl certificates when elasticsearch has tls enabled                                    | `NULL`                          |
 | `extraConfigOptions`                         | Additional options to propagate to all rules, cannot be `alert`, `type`, `name` or `index`                                    | `{}`                            |
 | `secretConfigName`                         | name of the secret which holds the Elastalert config. **Note:** this will overwrite the generated config and must contain a full working config                                      | `NULL`                            |
-| `secretRulesName`                         | name of the secret which holds the Elastalert rules. **Note:** this will overwrite the generated rules and must contain all rules in separate data fields (see values.yaml for an example)                                    | `NULL`                            |
+| `secretRulesName`                            | name of the secret which holds the Elastalert rules. **Note:** this will overwrite the generated rules. Must contain all rules in separate data fields (see values.yaml for an example) or `secretRulesMountsAsDirectory` must be enabled                                   | `NULL`                            |
 | `secretRulesList`                         | a list of rules to enable from the secret                                  | []                            |
+| `secretRulesMountsAsDirectory`               | Mount all rules from the secret. **Note:** this will disable the secretRulesList                                              | `false`                         |
```

Comment:
Should this comment be "will ignore" the secretRulesList, instead of "will disable"? As it's written someone could interpret that to mean any rule listed in `secretRulesList` will be _disabled_.

Reply:
I agree. Thank you @jertel